apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/nlog-template.xml

```diff
@@ -5,7 +5,7 @@
   <extensions>
     <add assembly="Google.Cloud.Logging.NLog" />
   </extensions>
-  
+
```

Comment:
Given that the only changes to this file are now whitespace, please revert it.

Reply:
Tried but Visual Studio is very persistent about making this change. Will try again.

Reply:
Success